streams/src/main/java/org/apache/kafka/streams/state/internals/KeyValueToKeyValueTimestampeIteratorAdapter.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.state.internals;
+
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.state.KeyValueIterator;
+
+import static org.apache.kafka.streams.state.TimestampedBytesStore.convertToTimestampedFormat;
+
+/**
+ * This class is used to ensure backward compatibility at DSL level between
+ * {@link org.apache.kafka.streams.state.TimestampedKeyValueStore} and
+ * {@link org.apache.kafka.streams.state.KeyValueStore}.
+ *
+ * @see KeyValueToKeyValueTimestampByteStoreAdapter
+ */
+class KeyValueToKeyValueTimestampeIteratorAdapter<K> implements KeyValueIterator<K, byte[]> {
```

Comment:
`KeyValueToKeyValueTimestampEIteratorAdapter` should probably be `KeyValueToKeyValueTimestampIteratorAdapter`

Reply:
Good catch!

Reply:
Ack. Actually, I will rename slighlty. Also forgot to rename the WindowProxy to WindowAdapter using the same pattern.